change/@ni-nimble-components-79631013-9204-4a60-85e7-c3e288944a9f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This change lets me make the middle of a link bold:
![image](https://github.com/ni/nimble/assets/1588923/9e11c265-da43-4bff-a62b-ed79bdccff76)

Which results in the following markdown:
![image](https://github.com/ni/nimble/assets/1588923/ca8016de-fbcd-47fa-ae55-5901d33f3a6d)

Which doesn't look right to me

Reply:
This is an issue with the current approach which requires complete implementation change in the PR. I am moving this PR back to draft and once found a proper fix, will make it ready for review, else will abandon this PR.